api/metrics/src/main/java/io/opentelemetry/api/metrics/LongUpDownCounterBuilder.java

```diff
@@ -5,14 +5,41 @@
 
 package io.opentelemetry.api.metrics;
 
+import java.util.function.Consumer;
+
 /** Builder class for {@link LongUpDownCounter}. */
-public interface LongUpDownCounterBuilder extends SynchronousInstrumentBuilder {
-  @Override
+public interface LongUpDownCounterBuilder {
+  /**
+   * Sets the description for this instrument.
+   *
+   * <p>Description stirngs should follw the instrument description rules:
```

Comment:
```suggestion
   * <p>Description strings should follow the instrument description rules:
```

Don't think I'll catch all of these in review, would be good if you can run a spell check

Reply:
Didn't realize I was missing one!  Added to visual studio, re-ran through all the main API